.appveyor.yml

```diff
@@ -1,36 +1,38 @@
-build: off
-version: dev-{build}
```

Comment:
Just wanted to see if it has something to do with cache not working. It doesn't so I'll restore it.